### PR TITLE
Add a SECURITY.md

### PR DESCRIPTION
**Pull request by mal-tee**

### CODE REVIEW

**Comment by zner0L** [comment]
Thank you! I am not sure if we maybe want another email address for this to distinguish it from all the spam? What do you think, @baltpeter?

**Comment by mal-tee** [comment]
He said he doesn't want to maintain a new email address/PGP key, see to our internal tracker: https://git.my-server.in/datenanfragen/vorstand/-/issues/36#note_5019 :D

**Comment by baltpeter** [comment]
> to distinguish it from all the spam

Pretty sure that would achieve the exact opposite of that: Spammers would rejoice over _yet another_ email address they can send their crap to.

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Security Policy
```
Comment (baltpeter) [inline .github/SECURITY.md:1]:
```suggestion
# Security policy
```

File: .github/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Thanking you for taking an interest in reporting a security vulnerability to us.
+
+You can reach us via `vorstand@datenanfragen.de`. You can use our [PGP key](https://www.datenanfragen.de/pgp/3E2B296B.asc) with the fingerprint `2E72 EA5B DDE3 1730 58D7  F87D A0C1 C012 3E2B 296B`.
```
Comment (baltpeter) [inline .github/SECURITY.md:5]:
```suggestion
You can reach us via [`vorstand@datenanfragen.de`](mailto:vorstand@datenanfragen.de). You can use our [PGP key](https://www.datenanfragen.de/pgp/3E2B296B.asc) with the fingerprint `2E72 EA5B DDE3 1730 58D7  F87D A0C1 C012 3E2B 296B`.
```

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**